### PR TITLE
Initial Page addition for the Venue Section.

### PR DESCRIPTION
**Pull request by SeriousHorncat**
This PR also includes updates to the READEME.md to install and run yarn locally on a Windows machine.

### CODE REVIEW

File: WeRockIT-ui/README.md [hunk 1]
```diff
@@ -37,3 +37,19 @@ yarn run test:unit
...
+
+    Note: If does not work as anticipated right after installation, restart the machine
+
+2. Install Yarn, a project dependency manager, using the instructions from `https://yarnpkg.com/lang/en/docs/install/#windows-stable`
+
+    Execute ```choco install yarn``` within a powershell terminal
+
+    Note: This will also install node.js as a yarn dependency
+
+    Note: Restart your computer for VSCode terminal to have yarn in the path to use within VSCode
```
Comment (RBangel) [inline WeRockIT-ui/README.md:52]: Do you really need to restart your system? Shouldn't you be able to just restart the powershell terminal or VScode terminal?

Comment (SeriousHorncat) [reply WeRockIT-ui/README.md:52]: I tried to do so and it didn't work. We could remove it though and let the next person who sets it up to see if it is necessary. This is also me trying to recall what I did a week ago and I could be remembering wrong.  I was surprised today that I was able to use it in the Powershell terminal that hooked up to VSCode today when it wasn't working when I first did the installation.

Comment (SeriousHorncat) [reply WeRockIT-ui/README.md:52]: I went ahead and removed it.  :D If it ends up not working we can add it back in.

**Review by RBangel — APPROVED** [review]
The only comment I have is one the README line. You commented that you removed it but It looks like it's still in the commit. I don't care nearly enough to block the PR with that, but was curious ;)

**Comment by SeriousHorncat** [comment]
> The only comment I have is one the README line. You commented that you removed it but It looks like it's still in the commit. I don't care nearly enough to block the PR with that, but was curious ;)

Alrighty! Will move forward and wrap up this PR then.

**Pull request merged**